### DISCUSSION TITLE
Approval checkboxes one Branch

### DISCUSSION THREAD

**Discussion by mikes02** [renovatebot/renovate#19227] [Request Help] [answered]
Is it possible to setup the issue dashboard to allow users to select what dependencies to update by having one checkbox per dependency and having renovate only create one branch/merge on the next run?  I have played around with various settings using dependencyDashboardApproval and groupName without success.  Any ideas?  Here is a snippet of my settings.

`  "dependencyDashboard": true,
  "dependencyDashboardApproval": true,
  "dependencyDashboardAutoclose": true,
  "packageRules": [
    {
      "matchDatasources": [
        "maven"
      ],
      "registryUrls": [
        "https://domain.com/repo"
      ],
      "groupName": "dependencies with minor/patch releases",
      "addLabels": [
        "Renovate-bot"
      ]
    },
    {
      "matchUpdateTypes": [
        "major"
      ],
      "enabled": false
    }
  ]`

**Comment by rarkins** [comment, marked as answer]
That type of feature is not supported
